cmd/export/main.go

```diff
@@ -17,37 +17,102 @@ package main
 
 import (
 	"context"
-	"log"
+	"fmt"
 	"net/http"
+	"time"
 
 	"go.opencensus.io/plugin/ochttp"
 
 	"github.com/google/exposure-notifications-server/internal/export"
 	"github.com/google/exposure-notifications-server/internal/logging"
 	_ "github.com/google/exposure-notifications-server/internal/observability"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
 	"github.com/google/exposure-notifications-server/internal/setup"
 )
 
 func main() {
 	ctx := context.Background()
 	logger := logging.FromContext(ctx)
 
+	if err := realMain(); err != nil {
+		logger.Fatal(err)
+	}
+}
+
+func realMain() error {
```

Comment:
if we're extracting this out for testability, it should probably go into /internal